docroot/modules/humsci/hs_blocks/hs_blocks.module

```diff
@@ -34,3 +34,17 @@ function hs_blocks_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function hs_blocks_preprocess_block__block_content(&$variables) {
+  /** @var \Drupal\block_content\Entity\BlockContent $block_entity */
+  $block_entity = $variables['content']['#block_content'];
```

Comment:
Is this always going to be available? Should you check that the array key's are set or do you want the errors if they are not?

Reply:
Since its preprocess on a block content entity it will always be there